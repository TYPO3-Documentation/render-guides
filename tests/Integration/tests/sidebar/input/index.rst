..  _rest-cards:
..  _rest-tabs:
..  _rest-confval:

=======
Sidebar
=======

..  sidebar:: reST content elements

    * :ref:`Cards <rest-cards>`
    * :ref:`Tabs <rest-tabs>`
    * :ref:`Configuration values <rest-confval>`

Displays a box to the right of the current paragraph. The box is mostly used
at the top of a page for quick navigation:
