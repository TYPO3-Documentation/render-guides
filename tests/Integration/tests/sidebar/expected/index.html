<!-- content start -->
                <section class="section" id="sidebar" data-rst-anchor="rest-confval">
            <a id="rest-confval"></a>
            <a id="rest-tabs"></a>
            <a id="rest-cards"></a>
            <h1>Sidebar<a class="headerlink" href="#sidebar" data-bs-toggle="modal" data-bs-target="#linkReferenceModal"  title="Reference this headline">&nbsp;<i class="fa-solid fa-link"></i></a></h1>
            <div class="sidebar">
    <p class="sidebar-title">reST content elements</p>

<ul>
    <li><a href="#rest-cards">Cards</a></li>
    <li><a href="#rest-tabs">Tabs</a></li>
    <li><a href="#rest-confval">Configuration values</a></li>
</ul>

</div>

    <p>Displays a box to the right of the current paragraph. The box is mostly used
at the top of a page for quick navigation:</p>

    </section>
        <!-- content end -->
